Makefile

```diff
@@ -0,0 +1,38 @@
+REBAR=rebar
```

Comment:
Please don't re-add a `Makefile`. Particularly not one using the old `rebar`. We use `rebar3` directly to maintain this project now.